### PR TITLE
Release 2.1.0

### LINKED ISSUE #40: new forge release
This works great for me with Debian 10, but those changes are not in the current version on the puppet forge.  Can we make a new release?

### PR DESCRIPTION
**Pull request by timdeluxe**
#### Pull Request (PR) description
New forge release of module

#### This Pull Request (PR) fixes the following issues
Fixes #40

### CODE REVIEW

**Comment by timdeluxe** [comment]
@bastelfreak or @ekohl 
Can i get an approval (and correct label and merge and release) here please?

**Review by bastelfreak — CHANGES_REQUESTED** [review]
I think before we do a release we should clean up the [metadata.json](https://github.com/voxpupuli/puppet-openvmtools/blob/master/metadata.json) and drop EOL Puppet 5 and EoL CentOS 6 from it

**Comment by timdeluxe** [comment]
Can't we do this release and do your suggestions in a 3.0.0 release as it is breaking?
Some still might support older stuff (like us currently) and this release might come in handy then.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,37 @@ All notable changes to this project will be documented in this file.
...
 These should not affect the functionality of the module.
 
-## [v2.0.1](https://github.com/voxpupuli/puppet-openvmtools/tree/v2.0.1) (2020-01-24)
+## [v2.1.0](https://github.com/voxpupuli/puppet-openvmtools/tree/v2.1.0) (2021-05-10)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-openvmtools/compare/v2.0.1...v2.1.0)
+
+**Breaking changes:**
+
+- Require puppet-epel over stahnma-epel [\#29](https://github.com/voxpupuli/puppet-openvmtools/pull/29) ([dhoppe](https://github.com/dhoppe))
```
Comment (ekohl) [inline CHANGELOG.md:13]: This is marked as a breaking change but this isn't a major version bump. That's a bit odd to me.

**Comment by ekohl** [comment]
FYI: I don't think removing the EOL things is that important. If you care about it, it's little effort to keep it in. You could even argue it's more work to remove it.

File: CHANGELOG.md (hunk 1 above)
Comment (timdeluxe) [reply CHANGELOG.md:13]: I just ran the github changelog generator. If that shall be changed either the label of that PR needs to change (i do not have permissions to do it) or we need to adapt the rake task to not identify that label as "breaking"...

Comment (ekohl) [reply CHANGELOG.md:13]: It's complicated whether it really is breaking or not. I can see the argument for both.

Comment (timdeluxe) [reply CHANGELOG.md:13]: Check here: https://github.com/voxpupuli/puppet-nodejs/pull/419 ...there it was not labeled with "backwards-incompatible" (causing the "breaking-change") and just with "doc" and it was merged by yourself. Maybe that helps on the decision?

Comment (timdeluxe) [reply CHANGELOG.md:13]: @ekohl Can we get progress here?

**Review by ekohl — APPROVED** [review]
Apologies for losing the notification in my inbox. I'm trying hard to trim it down. I'm ok with it but @bastelfreak explicitly requested a change so please check it with him as well.

**Comment by timdeluxe** [comment]
@ekohl Don't worry, i do have a similar problem of keeping track of all the PRs i raised.

@bastelfreak Can we plan a way forward? In the meantime i raised two other PRs (yet unmerged), which i would like to see in a release. My proposal would be to ditch this PR and create a new one for the 2.1.0 release. I still have the opinion that we should do a 2.1.0 with what got changed so far and dropping EOL systems in a 3.0.0 release then. I don't see the epel-dependency change as breaking, which would be worth a new major release, so i would propose to change the label on the epel-dependency change PR so that it does not pop up as breaking in the changelog (see also code discussion with ekohl in this PR).

**Comment by timdeluxe** [comment]
@bastelfreak Any update please?

**Comment by bastelfreak** [comment]
I rebased the changelog and submitted it again to https://github.com/voxpupuli/puppet-openvmtools/pull/47

**Pull request closed without merging**